### DISCUSSION TITLE
外网可以打开OpenWrt，但是不能打开控制面板

### DISCUSSION THREAD

**Discussion by SimonLiXi** [vernesong/OpenClash#2483] [Q&A]
也在防火墙设置了端口转发，是不是因为地址解析问题呢？还是其他版本正常呢？

**Comment by stacklix** [comment]
openwrt控制页面使用的端口和openclash控制页面使用的端口是不一样的

**Reply by SimonLiXi** [reply]
也试过其他的端口，一直没有成功，有没有一个成功的步骤?多谢

**Reply by stacklix** [reply]
你是有公网ip吗？ 是通过公网ip访问，还是通过内网穿透访问的

**Comment by wyychelsea** [comment]
最近碰到了同样的问题，我使用DDNSTO，外网可以访问openwrt，但是加上端口号9090后，外网不能访问openclash的控制面板

**Reply by SimonLiXi** [reply]
是的，不知道是不是设置问题。好像还是域名没法解析。

**Reply by wyychelsea** [reply]
> 是的，不知道是不是设置问题。好像还是域名没法解析。

老铁试试内网穿透神器ZeroTier

**Comment by rehack** [comment]
同样的问题

**Reply by SimonLiXi** [reply]
看看是不是把9090端口转发一下
